client/Android/Studio/freeRDPCore/src/main/java/com/freerdp/freerdpcore/presentation/SessionView.java

```diff
@@ -334,6 +334,8 @@ public boolean onSingleTapUp(MotionEvent e)
 			sessionViewListener.onSessionViewBeginTouch();
 			switch (e.getButtonState())
 			{
+      		         // If it's 0 then it's normal touch, or BUTTON_PRIMARY then Samsung DeX or any other physical mouse left click.
+				case 0:
```

Comment:
So, you tell android docs are wrong?
see `https://developer.android.com/reference/android/view/MotionEvent.html#getButtonState()`

I might be mistaken, but if you want support for physical devices you should listen for these events separately.